src/components/pages/MapPage/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from "react";
+
+import axios from "axios";
+import GoogleMapReact from "google-map-react";
+import styled from "styled-components";
+
+import theme from "../../../theme/theme";
+import { Icon } from "../../atoms";
+import { MapTemplate } from "../../templates";
+
+const StyledIcon = styled(Icon)`
+  color: ${(props) => props.color};
+`;
+
+function MapPage() {
+  const [defaultProps, setDefaultProps] = useState(null);
+  const [zoomLevel, setZoomLevel] = useState(14);
+  const [boundary, setBoundary] = useState({});
+  const [feedLocation, setFeedLocation] = useState([]);
+
+  function spreadFeeds() {
+    return feedLocation.map((feed) => {
+      const { _id, image, cleaned, coordinates } = feed;
+      const [longitude, latitude] = coordinates;
+
+      if (cleaned) {
+        return <StyledIcon key={_id} lat={latitude} lng={longitude} icon="leaf" color={theme.colors.green_1} />;
+      }
+
+      return <StyledIcon key={_id} lat={latitude} lng={longitude} icon="trashCanFill" color={theme.colors.red} />;
```

Comment:
```suggestion
    const iconType = cleaned ? "leaf" : "trashCanFill";
    const color = cleaned ? theme.colors.green_1 : theme.colors.red;

    return <StyledIcon key={_id} lat={latitude} lng={longitude} icon={iconType} color={color} />;
```

중복되는 코드를 정리할 수 있을 것 같습니다~!

Reply:
더 깔끔해 보이네요👍  수정진행하겠습니다.